FluentUI.Demo/src/androidTest/java/com/microsoft/fluentuidemo/demos/V2BottomDrawerUITest.kt

```diff
@@ -241,4 +248,18 @@ class V2BottomDrawerUITest : BaseTest() {
         }
         closeCheckForVerticalDrawer()
     }
+    @Test
+    fun testNonExpandableAndExpandDrawerClicked() {
+        // Set expandable = false
+        composeTestRule.onNodeWithText("Expandable", useUnmergedTree = true).performClick()
```

Comment:
Use String Id to retrieve string. This is required to run test in other locale.

Reply:
Done.